CefGlue.Common/CefGlue.Common.csproj

```
@@ -31,6 +31,7 @@
     <ItemGroup Condition="'$(Platform)' == 'x64'">
         <PackageReference PrivateAssets="None" Include="chromiumembeddedframework.runtime.win-x64" />
         <PackageReference PrivateAssets="None" Include="cef.redist.osx64" />
+        <PackageReference PrivateAssets="None" Include="cef.redist.linux64" />
```

created a new [package](https://www.nuget.org/packages/cef.redist.linux.x64/117.2.4) with this name
```suggestion
        <PackageReference PrivateAssets="None" Include="cef.redist.linux-x64" />
```